.rubocop.yml

```diff
@@ -1,14 +1,16 @@
-require:
+plugins:
+  - rubocop-performance
   - rubocop-rails
   - rubocop-rspec
   - rubocop-rspec_rails
   - rubocop-capybara
   - rubocop-factory_bot
 
-plugins:
-  - rubocop-performance
-
 AllCops:
+  TargetRubyVersion: 3.4
+  TargetRailsVersion: 7.1
+  ExtraDetails: true
+  DisplayStyleGuide: true
```

Comment:
Extra details and style guide link are handy for figuring out what the rule actually wants you to do